pages/registration/financial-aid.vue

```diff
@@ -1,11 +1,191 @@
 <template>
-    <p>Registration financial aid</p>
+    <i18n-page-wrapper>
+        <core-h1 :title="$t('title')"></core-h1>
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('coreValueTitle') }}</p>
+            </template>
+            <template #right-col>
+                <p class="content paragraph">
+                    <ext-link
+                        href="https://pyfound.blogspot.com/2017/10/psfs-october-board-meeting.html"
+                        highlight
+                        underline
+                    >
+                        {{ $t('everybodyContributes') }}
+                    </ext-link>
+                    {{ $t('coreValueContent')[0] }}
+                </p>
+                <p class="content paragraph">
+                    {{ $t('coreValueContent')[1] }}
+                </p>
+            </template>
+        </two-col-wrapper>
+        <core-hr></core-hr>
+        <two-col-wrapper
+            v-for="(field, i) in $t('plainTextFields')"
+            :key="`field_${i}`"
```

Comment:
`field_<index>` is quite common and could be duplicated in other pages. Please follow our current naming convention: `<page name>_<component/section name>_<index>`.